ml-proto/test/memory.wast

```diff
@@ -95,7 +95,7 @@
     (loop
       (if
         (i32.eq (get_local 0) (i32.const 0))
-        (br 1)
+        (br 2)
```

Comment:
This pull is for `return` - why did these `br`s change?

And where does 2 go? the only `block`s or `loops` above this `br` are the one `loop`, which would take indexes 0 and 1?